client/package.json

```diff
@@ -5,13 +5,18 @@
   "dependencies": {
     "react": "^16.7.0",
     "react-dom": "^16.7.0",
-    "react-scripts": "2.1.2"
+    "react-scripts": "2.1.2",
+    "tachyons": "^4.11.1",
+    "tachyons-cli": "^1.3.2"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject",
+    "css": "tachyons src/css/app.css > src/index.css --minify",
+    "prestart": "npm run css",
```

Comment:
Does it work for you ? I get this weird error 'unknown browsers dead' :

<img width="1280" alt="screen shot 2018-12-28 at 11 28 40" src="https://user-images.githubusercontent.com/883348/50512745-05c8e400-0a94-11e9-9054-d07990015c6e.png">

It seems related with using a deprecated version of browserslist.

To be completely transparent, I'm not really sure we need to use the `tachyons-cli` package at all. can you just try to remove it and remove this `css` script thing ?

Reply:
According to this issue https://github.com/tachyons-css/cli/issues/22, in `client/package.json` I had to delete this part 
`
"browserslist": [
  ">0.2%",
  "not dead",
  "not ie <= 11",
  "not op_mini all"
]
`

Reply:
Deleted tachyons-cli + script CSS, works fine 😃